tests/unit/dialog/common-deprecated.js

```diff
@@ -34,6 +34,7 @@ common.testWidget( "dialog", {
 		resizable: true,
 		show: null,
 		title: null,
+		uiDialogTitleTagName: "<span>",
```

Comment:
Yes, this needs to be added here as well. 👍🏻 

Most modules have regular & deprecated test modes and run tests in both. The deprecated one loads `common-deprecated.js` instead of `common.js` and it loads an additional `deprecated.js` file:
https://github.com/jquery/jquery-ui/blob/02a6e6bcb004696b21324ef14c07374a56d7cd02/tests/lib/bootstrap.js#L155-L164